### PR TITLE
FEM-2751 cont

### PR DESCRIPTION
**Pull request by giladna**

### CODE REVIEW

File: documentation/player-portal/android/player-config-android.md [hunk 1]
```diff
@@ -61,7 +87,7 @@ The PKMediaEntry object contains a list of "PKMediaSources". All sources relate
 
 To learn more, see [PKMediaSource](https://github.com/kaltura/playkit-android/blob/develop/playkit/src/main/java/com/kaltura/playkit/PKMediaSource.java)
 
-#### Manually a Create Media Source  
+#### Manually Create Media Source
```
Comment (jessp01) [inline documentation/player-portal/android/player-config-android.md]: Should read: "Manually create a media source"

File: documentation/player-portal/android/player-config-android.md [hunk 2]
```diff
@@ -35,14 +50,25 @@ Once you have a PKMediaEntry ready, you can build the player configuration and p
...
+
+The PkMediaEntry holds information gathered from the media API's and needed for the player and plugins to function properly, such as the URL to play, the DRM data, duration and the type of the media.
+
+This object usually will be created using one of the media providers in case Kaltura's BE is used for hosting your media files.
 
 Additional information includes:
 
 * String id - correlates to the media/entry id
-* long duration - the media duration in seconds
+* String name - meida name
```
Comment (jessp01) [inline documentation/player-portal/android/player-config-android.md]: Typo.

File: documentation/player-portal/android/player-config-android.md [hunk 3]
```diff
@@ -35,14 +50,25 @@ Once you have a PKMediaEntry ready, you can build the player configuration and p
 
 ### About the PKMediaEntry  
 
-The PkMediaEntry holds information gathered from the media details and needed for the player, such as the URL to play, the DRM data, and duration.
+#### NOTE:
+
+* Building the `PKMediaEntry`  manullay is more relevant for customers which do not use Kaltura BE to store their content.
```
Comment (jessp01) [inline documentation/player-portal/android/player-config-android.md]: typo

File: documentation/player-portal/android/player-config-android.md [hunk 4]
```diff
@@ -35,14 +50,25 @@ Once you have a PKMediaEntry ready, you can build the player configuration and p
...
+ 
+* Once content is stored in Kaltura's BE you can use our `Media Providers` to fetch the PKMediaEntry 
+
+The PkMediaEntry holds information gathered from the media API's and needed for the player and plugins to function properly, such as the URL to play, the DRM data, duration and the type of the media.
```
Comment (jessp01) [inline documentation/player-portal/android/player-config-android.md]: Not sure what you mean by "the media API's"...

File: documentation/player-portal/android/player-config-android.md (hunk 2 above)
Comment (giladna) [reply documentation/player-portal/android/player-config-android.md]: done

File: documentation/player-portal/android/player-config-android.md (hunk 1 above)
Comment (giladna) [reply documentation/player-portal/android/player-config-android.md]: fixed

File: documentation/player-portal/android/player-config-android.md (hunk 4 above)
Comment (giladna) [reply documentation/player-portal/android/player-config-android.md]: fixed

**Pull request merged**